testdata/generate_pyiceberg.py

```diff
@@ -0,0 +1,55 @@
+"""
+How to run this script:
+======================
+
+$ python3 -m venv venv
+$ source venv/bin/activate
+$ pip install "pyiceberg[pyarrow,sql-sqlite]"
+$ pip install botocore
+$ python ./testdata/generate_pyiceberg.py
```

Comment:
I  think the data set ends up being pretty big, and while I don't think it _really_ matters, I  think it'd be good to avoid just committing lots of test data to the repo _just cause_, when we have other options (e.g. uploading the data to the test bucket, or generating as part of a fixture in one of the pytests).

Reply:
Yeah, thinking the same thing.